backend/src/services/itinerary_generator.py

```
@@ -123,35 +126,43 @@ async def create_itinerary(self, extracted_data: Dict) -> Dict:
         
         return itinerary
     
-    def _infer_destination(self, flights: List[Dict], hotels: List[Dict]) -> str:
+    def _infer_destination(self, flights: List[Dict], 
+                          hotels: List[Dict]) -> str:
         """Infer destination from flights and hotels"""
 
         print(f"[DESTINATION] 🎯 Inferring destination from extracted data...")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)